api/src/Security/Authenticator/HttpBearerAuthenticator.php

```diff
@@ -0,0 +1,159 @@
+<?php
+
+namespace App\Security\Authenticator;
+
+use App\Security\Authenticator\Passport\Badge\TokenBadge;
+use App\Security\Authenticator\Passport\TokenPassport;
+use Lcobucci\JWT\Configuration;
+use Lcobucci\JWT\Encoding\CannotDecodeContent;
+use Lcobucci\JWT\Exception;
+use Lcobucci\JWT\Signer;
+use Lcobucci\JWT\Signer\Key\FileCouldNotBeRead;
+use Lcobucci\JWT\Signer\Key\InMemory;
+use Lcobucci\JWT\Validation\Constraint\SignedWith;
+use Psr\Log\LoggerInterface;
+use RuntimeException;
+use Symfony\Component\HttpClient\HttpClient;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Security\Core\Authentication\Token\TokenInterface;
+use Symfony\Component\Security\Core\Exception\AuthenticationException;
+use Symfony\Component\Security\Core\User\UserProviderInterface;
+use Symfony\Component\Security\Http\Authenticator\AuthenticatorInterface;
+use Symfony\Component\Security\Http\Authenticator\Passport\Passport;
+use Symfony\Component\Security\Http\Authenticator\Token\PostAuthenticationToken;
+use Symfony\Component\Security\Http\EntryPoint\AuthenticationEntryPointInterface;
+use Symfony\Contracts\HttpClient\Exception\ExceptionInterface;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+
+/**
+ * @author Vincent Chalamon <vincentchalamon@gmail.com>
+ *
+ * @final
```

Comment:
some classes are marked `@final` others with the final T_FINAL token. I suggest to be homogeneous on this.

Reply:
I followed what is currently done in Symfony: some are marked `@final`, others T_FINAL